### PR TITLE
Improve documentation of game messages

### PR DESCRIPTION
**Pull request by Garanas**
Related to: https://github.com/FAForever/fa/pull/6479

Improves the documentation of some of the game messages.

### CODE REVIEW

File: server/gameconnection.py [hunk 1]
```diff
@@ -547,6 +571,19 @@ async def handle_game_full(self):
         """
         pass
 
+    async def handle_established_peers(self, peerId: str, peerConnectedTo: str):
```
Comment (Askaholic) [inline server/gameconnection.py]: Params should be `snake_case`

File: server/gameconnection.py [hunk 2]
```diff
@@ -547,6 +571,19 @@ async def handle_game_full(self):
...
+        """
+        Send by the lobby when a peer connected to another peer. Message can be send multiple times, by multiple connections.
+
+        # Params
+        - peerId: The identifier of the peer that this connection received the message from
```
Comment (Askaholic) [inline server/gameconnection.py]: Add backticks so it matches the docstring for teamkill_happened. These get rendered as Markdown in our docs.
```suggestion
        - `peerId`: The identifier of the peer that this connection received the message from
```

File: server/gameconnection.py [hunk 3]
```diff
@@ -547,6 +571,19 @@ async def handle_game_full(self):
...
+        - peerConneectedTo: The list of identifiers that the given peer established a connection to. Is a list stored in a string, separated by spaces. As an example: "1321 22 33 43221 2132"
+        """
+
+        # peerConnectedToList = peerConnectedTo.split(" ")
```
Comment (Askaholic) [inline server/gameconnection.py]: You can remove the commented out code. I think I’ll be able to figure that one out :)
```suggestion
```

**Review by Askaholic — COMMENTED** [review]
Cool, thanks! I hope to find some time this weekend to implement the established peers.

What are we expecting the server to do with the launch status information? If we’re just ignoring most of it you could also just add the possible options in the docstring and get rid of the empty if chain.

**Comment by Garanas** [comment]
The launch status can help you understand what is happening to a player/connection. For example - a player that has the status `Ready` is connected to all its peers and is ready to launch. Such a player should never receive a (time) penalty if the lobby does not start for some reason.

edit: perhaps 'launch status' is the wrong term to use here.

**Comment by Askaholic** [comment]
I'm just going to combine this with #1028 for simplicity. Thanks for making the initial changes!

**Pull request closed without merging**